test/e2e/gateway_api_test.go

```diff
@@ -191,6 +195,105 @@ func testGatewayAPIObjects(t *testing.T) {
 	}
 }
 
+// testGatewayAPIManualDeployment verifies that Istio's manual deployment is not
+// enabled.  When manual deployment is enabled, then Istio allows a gateway use
+// to another gateway's service by specifying that gateway's service in
+// spec.addresses.  This results in behavior similar to gateway listener
+// merging, which we do not want to allow at this time.  Instead, we expect
+// Istio to provision a service for this specific gateway, even if it specifies
+// spec.addresses.
+func testGatewayAPIManualDeployment(t *testing.T) {
+	gatewayClass, err := createGatewayClass("openshift-default", "openshift.io/gateway-controller")
```

Comment:
```suggestion
	gatewayClass, err := createGatewayClass("openshift-default", "openshift.io/gateway-controller/v1")
```

Reply:
Very odd - I don't understand how the test works without the correct controllerName.

Reply:
I haven't rerun the tests since #1202 merged.

Reply:
Fixed in  https://github.com/openshift/cluster-ingress-operator/compare/1f0444b996dfff19d6118aadb7309bc8e1293fa8..6734479ebfc5bbb83897bc51f7d7ca580f783e2b.